code/_globalvars/lists/objects.dm

```diff
@@ -41,6 +41,7 @@ GLOBAL_LIST_EMPTY(alert_consoles)			// Station alert consoles, /obj/machinery/co
 GLOBAL_LIST_EMPTY(xeno_resin_silo_turfs)
 GLOBAL_LIST_EMPTY(xeno_resin_silos)
 GLOBAL_LIST_EMPTY(xeno_turret_turfs)
+GLOBAL_LIST_EMPTY(xeno_turret_tiles)
```

Comment:
make a list of the turrets instead of the tiles

Reply:
That just happens to be what I named the list. And that's technically what it is-a list of tiles containing turrets, since it adds the location to the list. Are you saying you want me to rename the list?

Reply:
No I want you to change it to a list of turrets because a list like that is much more useful in the lng run and has 0 impact on the rest of your code